src/extensionShared.ts

```diff
@@ -11,10 +11,42 @@
 import vscode from 'vscode'
 import globals from './shared/extensionGlobals'
 import { join } from 'path'
+import { Commands } from './shared/vscode/commands2'
+import { documentationUrl, githubCreateIssueUrl, githubUrl } from './shared/constants'
+import { getIdeProperties, aboutToolkit } from './shared/extensionUtilities'
+import { telemetry } from './shared/telemetry/telemetry'
+import { openUrl } from './shared/utilities/vsCodeUtils'
 
 export function initializeManifestPaths(extensionContext: vscode.ExtensionContext) {
     globals.manifestPaths.endpoints = extensionContext.asAbsolutePath(join('resources', 'endpoints.json'))
     globals.manifestPaths.lambdaSampleRequests = extensionContext.asAbsolutePath(
         join('resources', 'vs-lambda-sample-request-manifest.xml')
     )
 }
+
+export function activateGlobalCommands(extensionContext: vscode.ExtensionContext) {
```

Comment:
Nit all commands are "global" more or less.
```suggestion
export function registerCommands(extensionContext: vscode.ExtensionContext) {
```